registry/search.go

```diff
@@ -74,6 +62,8 @@ func (s *Service) Search(ctx context.Context, searchFilters filters.Args, term s
 				continue
 			}
 		}
+		// "is-automated" is deprecated and will always be false
+		result.IsAutomated = false //nolint:staticcheck  // ignore SA1019 (field is deprecated)
```

Comment:
If it's deprecated and always false in Hub results, why do we need to explicitly set it to false ourselves?

Reply:
IIUC, on Hub side this is not always false (yet), it's just deprecated.

But @thaJeztah  might have more context here